mlos_bench/mlos_bench/config/services/remote/azure/scripts/setup-rg/rg-template.json

```diff
@@ -0,0 +1,346 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "projectPrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Prefix used for naming resources."
+            },
+            "defaultValue": "mlos"
+        },
+        "vmSKU": {
+            "type": "string",
+            "defaultValue": "Standard_D2s_v3",
+            "metadata": {
+                "description": "VM SKU"
+            }
+        },
+        "vmAdminUsername": {
+            "type": "string"
+        },
+        "sshPublicKeys": {
+            "type": "array"
+        },
+        "storageAccountLocations": {
+            "type": "array",
+            "defaultValue": [
+                "[resourceGroup().location]"
+            ]
+        },
+        "fileShareName": {
+            "type": "string",
+            "defaultValue": "mlos-file-share"
+        },
+        "vmLocation": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]"
+        },
+        "vmSshPort": {
+            "type": "string",
+            "defaultValue": "22"
+        },
+        "vmSshSourceAddressPrefix": {
+            "type": "string"
+        },
+        "storageSku": {
+            "type": "string",
+            "defaultValue": "Standard_LRS",
+            "allowedValues": [
+                "Premium_LRS",
+                "Premium_ZRS",
+                "Standard_GRS",
+                "Standard_GZRS",
+                "Standard_LRS",
+                "Standard_RAGRS",
+                "Standard_RAGZRS",
+                "Standard_ZRS"
+            ]
+        }
+    },
+    "functions": [],
+    "variables": {
+        "storageAccountName": "[toLower(concat(replace(parameters('projectPrefix'), '-', '')))]",
+        "vmName": "[concat(parameters('projectPrefix'), '-vm')]",
+        "networkInterfaceName": "[concat(variables('vmName'), '-ni-', uniqueString(resourceGroup().id))]",
+        "vnetName": "[concat(variables('vmName'), '-vnet')]",
```

Comment:
Let's name the vnet something less tied to just the single VM name.
That way we can add more VMs/services (additional schedulers, frontends, storage, etc.) to it in the future if necessary.

Reply:
> This will require fixups throughout the rest of this config.

Reply:
Agreed, changed these to the project prefix instead.